python/lsst/meas/extensions/astrometryNet/anetBasicAstrometry.py

```diff
@@ -300,7 +300,7 @@ def _getImageParams(self, exposure=None, bbox=None, wcs=None, filterName=None, w
         """Get image parameters
 
         @param[in] exposure  exposure (an afwImage.Exposure) or None
-        @param[in] bbox  bounding box (an afwGeom.Box2I) or None; if None then bbox must be specified
+        @param[in] bbox  bounding box (an geom.Box2I) or None; if None then bbox must be specified
```

Comment:
Should this be `lsst.geom.Box2I`?

Reply:
Yes, but since (1) these docstrings all need fixing up when they are converted to numpydoc and (2) they already do it wrong by using `afwImage` in the line above, I think I'm better off not fixing these on this pass.

Reply:
Not a problem. Leave these as they are, then.